go/tao/datalog_guard.go

```diff
@@ -618,7 +624,122 @@ func (g *DatalogGuard) retract(f auth.Form) error {
 	return nil
 }
 
+func max(args ...int) int {
+	m := 0
```

Comment:
This function is overkill. It should just be a standard 2-arg max function. See below.